.github/workflows/test.yml

```diff
@@ -33,3 +33,78 @@ jobs:
 
       - run: pip3 install hatch
       - run: CONFIG_ROOT_DIR=`pwd`"/dags" hatch run tests.py3.12-2.10:static-check
+
+  Run-Unit-Tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
```suggestion
```
Decide if we want to keep this.

Reply:
I think it's usually useful to see all failures, so it seems a good idea to keep this line. WDYT?

Reply:
yes, I feel keeping it helps in the developer journey :)